src/js/index.js

```diff
@@ -164,6 +165,7 @@
       var isTop = e && e.target && e.target.scrollingElement && e.target.scrollingElement.scrollTop === 0
       if ((e && (e.eventPhase === 0 || e.currentTarget === null)) || isTop) {
         buildHtml.updateToc(headingsArray)
+        !options.disableTocScrollSync && updateTocScroll(options)
```

Comment:
It should also call the updateToScroll on the onScroll on the `scrollContainer` as tocbot is already setuping this up and has the knowledge in the init (to maintain bi-directional update)

Reply:
oops, nevermind, it is already in the scroll listener

Reply:
So not sure about the code in case of bi-directional update then, as my code is running `updateTocScroll` on the onscroll event, while the here it is running only `scrollEndCallback`, which seems to be triggered only when you explicitly click on the toc, not when you just scroll no?

Reply:
In that case, I would move `!options.disableTocScrollSync && updateTocScroll(options)` along the first `buildHtml.updateToc` at the beginning of the `_scrollListener` function

Reply:
I updated the PR and moved that line up in the scroll listener. Good call! :)